<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DealUpdateDialog</class>
 <widget class="QDialog" name="DealUpdateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Редактирование договора</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Сектор:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Договор:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="dealEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Тема:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <widget class="QLineEdit" name="themeEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Предмет:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLineEdit" name="itemEdit"/>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="saveButton">
     <property name="toolTip">
      <string>Сохранить изменения</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="2">
    <widget class="QPushButton" name="rejectButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="MyComboBox" name="secCombo"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Организация:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="MyComboBox" name="orgCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Дата заключения:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="MyDateEdit" name="dateEdit" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyDateEdit</class>
   <extends>QWidget</extends>
   <header location="global">mydateedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
